src/AltinnCore/Designer/Controllers/ServiceDevelopmentController.cs

```diff
@@ -132,9 +140,14 @@ public IActionResult SaveServiceFile(string org, string service, FileEditorMode
             return StatusCode(200);
         }
 
-        private ActionResult GetImplementationFiles(string org, string service)
+        private ActionResult GetImplementationFiles(string org, string service, bool calculation)
         {
             List<AltinnCoreFile> files = _repository.GetImplementationFiles(org, service);
+            if (calculation)
```

Comment:
maybe rewrite so that it doesnt fetch both files if calculations?

Reply:
so:
files = null;
if(calc){
files = getCalcFiles
} else {
files = getImplFiles
}

Or maybe create a new method GetCalculationFiles so the method does not have to care about the calculation flag?

Reply:
Could you please explain what your concern is?
The function should return all files in the calculation folder.